src/pages/Landing.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import "./Landing.css";
+import ScrollAnimation from 'react-animate-on-scroll';
+import "animate.css/animate.min.css";
+import sproul from './assets/sproul.svg';
+import ball1 from './assets/ball1.svg';
+import ball2 from './assets/ball2.svg';
+import ball3 from './assets/ball3.svg';
+import ball4 from './assets/ball4.svg';
+import screen1 from './assets/screen1.svg';
+import screen2 from './assets/screen2.svg';
+import screen3 from './assets/screen3.svg';
+import mock1 from './assets/mock1.png';
+import mock2 from './assets/mock2.png';
+import mock3 from './assets/mock3.png';
+
+const Landing = () => {
+  return (
+      <div className="landing">
+        <div className="header">
+          <a href="/" class="logo">sproul.club</a>
```

Comment:
We will probably want to use <Link>s instead of <a> because I am pretty sure <a> refresh the page when Link just renders the other component